Hmac.js

```diff
@@ -66,12 +105,45 @@ export class Hmac {
    * @param {Uint8Array} options.data - The data to sign as a Uint8Array.
    * @param {string} options.signature - The base64url-encoded signature
    *   to verify.
+   * @param {boolean} options.useCache - Enable the use of a cache.
```

Comment:
Is this the right doc syntax?

```suggestion
   * @param {boolean} [options.useCache=true] - Enable the use of a cache.
```

Reply:
Updated jsdoc to reflect default parameter.